src/super_gradients/examples/dota_prepare_dataset/dota_prepare_dataset.py

```diff
@@ -0,0 +1,47 @@
+import argparse
+from pathlib import Path
+
+import cv2
+from super_gradients.training.datasets import DOTAOBBDataset
+
+
+def main():
```

Comment:
An example is good.
However I think it's best to add some basic instructions like we have for the rest of our datasets in our src/super_gradients/training/datasets/Dataset_Setup_Instructions.md

Side note - we should probably move it to the docs directory (I guess that might be the reason you missed it).

Reply:
Fixed in 619e26bfa69c20fb83a76a10c113f618d185844c